client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/network/TransportFrameDecoderWithBufferSupplier.java

```diff
@@ -144,6 +148,9 @@ public void channelRead(ChannelHandlerContext ctx, Object data) {
           }
         }
       }
+    } catch (Exception e) {
```

Comment:
Do this need to notify the server?

Reply:
NO. Because read data won't expect rpc response. And other RPC decode failure means that the data is broken. We might can not send a response to the server.